### DISCUSSION TITLE
Verifing GPU is utilised

### DISCUSSION THREAD

**Discussion by vmatekole** [openai/whisper#603] [Q&A] [answered]
New to ML and Whisper. Spun up a GPU VM with my provider as I am transcribing podcasts, multiple hrs long. 
How can I verify that Whisper is utilizing GPU not CPU? 
Finally, can Whisper utilise multiple GPUs? Or are they additional efforts required or customisations to whisper?

**Comment by luigisaetta** [comment, marked as answer]
well, you should check that PyTorch can use GPU.
try this snippet of code.

```python
import torch
print(torch.cuda.is_available())
```

and it must return True. Otherwise, the environment is not configured to use CUDA.
